README.md

```diff
@@ -852,6 +852,50 @@ If you want the server to enforce and require client certificates as well via th
 > ./gnatsd --tlsverify --tlscert=./test/configs/certs/server-cert.pem --tlskey=./test/configs/certs/server-key.pem --tlscacert=./test/configs/certs/ca.pem
 ```
 
+#### TLS Authorization
+
+If `verify_and_map` is set as part of the TLS configuration, mutual TLS will be enabled and the certificate provided provided by a client will be used to authorize and map it permissions.
```

Comment:
provided * 2.

, mutual -> , client certificates will be required and mutual tls will be enable..